spec/src/main/asciidoc/core/events.asciidoc

```diff
@@ -296,22 +296,27 @@ An event parameter may have multiple qualifiers.
 public void afterDocumentUpdatedByAdmin(@Observes @Updated @ByAdmin Document doc) { ... }
 ----
 
-Then this observer method will only be notified if all the observed event qualifiers are specified when the event is fired:
+Then this observer method will be notified if the set of observer qualifiers is a subset of the fired event's qualifiers:
 
 [source, java]
 ----
 @Inject Event<Document> documentEvent;
 ...
-documentEvent.select(new UpdatedQualifier(), new ByAdminQualifier()).fire(document);
+documentEvent.select(new UpdatedQualifier(), new ByAdminQualifier(), new ClarificationQualifier()).fire(document);
 ----
 
-Other, less specific, observers will also be notified of this event:
+In the above example the event is fired with `@ByAdmin`, `@Updated`, `@Clarification` and `@Any` qualifiers. `@Any` is added automatically as described in <<event_types_and_qualifier_types>>. The observer qualifiers are `@Updated`, `@ByAdmin` and again `@Any`.
```

Comment:
Just to clarify - I am aware that in practice, the event has `@Default` qualifier as well (derived from section 2.3.1 Built-in qualifier types in spec), but I would rather avoid going into that here as it would probably introduce more confusion.